src/services/api/handlers/submit.ts

```diff
@@ -92,7 +92,7 @@ export const submit = async (event: APIGatewayEvent) => {
 
     const result = await sql.query(query);
     console.log(result);
-    if (body.authority == Authority["1915b"]) {
+    if (body.authority == Authority["1915b"] || Authority.CHIP_SPA) {
```

Comment:
This reads to me as "if body authority loose equals 1915b, or Authority.CHIP_SPA exists", but I think you're looking for "if body authority loose equals 1915b or Authority.CHIP_SPA".

```typescript
if (body.authority == Authority["1915b"] || body.authority == Authority.CHIP_SPA) {...}
// or my preferred way
if ([Authority["1915b"], Authority.CHIP_SPA].includes(body.authority)) {...}
```